src/sentry/api/serializers/models/alert_rule_trigger_action.py

```diff
@@ -20,14 +20,18 @@ def human_desc(self, action):
             return "Send a Microsoft Teams notification to " + action.target_display
         elif action.type == action.Type.SENTRY_APP.value:
             return "Send a notification via " + action.target_display
+        elif action.type == action.Type.OPSGENIE.value:
+            return "Send an Opsgenie notification to " + action.target_display
 
     def get_identifier_from_action(self, action):
         if action.type in [
             AlertRuleTriggerAction.Type.PAGERDUTY.value,
             AlertRuleTriggerAction.Type.SENTRY_APP.value,
         ]:
             return int(action.target_identifier)
-
+        if action.type == AlertRuleTriggerAction.Type.OPSGENIE.value:
+            # return team ID
+            return action.target_identifier
```

Comment:
It may help to give an example here as to why its different than the `SENTRY_APP` and `PAGERDUTY` section above. Those are integers, but these are longer IDs right?